### PR TITLE
[opt](thrift)update thrift to support pushing limit to local Agg

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by englefly** [comment]
run buildall

File: gensrc/thrift/PlanNodes.thrift [hunk 1]
```diff
@@ -904,7 +904,7 @@ struct TAggregationNode {
   7: optional list<TSortInfo> agg_sort_infos
   8: optional bool is_first_phase
   9: optional bool is_colocate
-  // 9: optional bool use_fixed_length_serialization_opt
+  10: optional TSortInfo agg_sort_info_by_group_keya
```
Comment (morrySnow) [inline gensrc/thrift/PlanNodes.thrift:907]:
```suggestion
  10: optional TSortInfo agg_sort_info_by_group_key
```

**Comment by englefly** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**